components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -618,15 +619,15 @@ TEST_F(PublisherInfoDatabaseTest, GetPanelPublisher) {
    * Publisher ID is missing
    */
   ledger::ActivityInfoFilter filter_1;
-  EXPECT_EQ(publisher_info_database_->GetPanelPublisher(filter_1),
+  EXPECT_EQ(publisher_info_database_->GetPanelPublisher(filter_1.Clone()),
```

Comment:
`filter_1.Clone()` should use `std::move(filter_1)` as `filter_1` is no longer used

Reply:
fixed